modules/samples/virtual_firealarm/src/org.wso2.carbon.device.mgt.iot.sample.virtual.firealarm.service.impl/src/main/java/org/wso2/carbon/device/mgt/iot/sample/virtual/firealarm/service/impl/util/VirtualFireAlarmMQTTSubscriber.java

```diff
@@ -68,10 +64,14 @@ protected void postMessageArrived(String topic, MqttMessage message) {
 			if(log.isDebugEnabled()) {
 				log.debug("MQTT Subscriber: Published data to DAS successfully.");
 			}
-		} else {
-			log.info("MQTT: Reply Message [" + message.toString() + "] topic: [" + topic + "]");
-		}
-	}
+        } else if (message.toString().contains("TEMPERATURE")) {
+            log.info("MQTT: Reply Message [" + message.toString() + "] topic: [" + topic + "]");
+            float temperature = Float.parseFloat(message.toString().split(":")[1]);
+            DataHolder.getThisInstance().setTemperature(deviceId, temperature, Calendar.getInstance().getTimeInMillis());
+        } else {
+            log.info("MQTT: Message [" + message.toString() + "] topic: [" + topic + "]");
```

Comment:
avoid unnecessary info messages as much as possible. These type of regular messages would be mostly appropriate in a debugging context.